packages/dumb/src/provider/trends.provider.hook.tsx

```diff
@@ -25,21 +25,34 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { useState } from 'react'
+import { useRef, useState } from 'react'
 import { type TrendsDisplayFlags } from '../models/trends-display-flags.model'
 import { DisplayFlag } from '../models/enums/display-flag.enum'
 import { type CbgDateTrace } from '../models/cbg-date-trace.model'
-import { type CbgPositionData } from '../models/cbg-position-data.model'
+import {
+  type CbgDateTraceYPositions,
+  type CbgPositionData,
+  type CbgSliceYPositions
+} from '../models/cbg-position-data.model'
 import { type FocusedCbgDateTrace } from '../models/focused-cbg-date-trace.model'
+import { type RangeSegmentSlice } from '../models/enums/range-segment.enum'
+import { type CbgSlice } from '../models/cbg-slice.model'
+import { type FocusedCbgSlice } from '../models/focused-cbg-slice.model'
 
 export interface TrendsContextResult {
   displayFlags: TrendsDisplayFlags
-  focusCbgDateTrace: (cbgDateTrace: CbgDateTrace, position: CbgPositionData) => void
+  focusCbgDateTrace: (cbgDateTrace: CbgDateTrace, position: CbgPositionData<CbgDateTraceYPositions>) => void
   focusedCbgDateTrace: FocusedCbgDateTrace | undefined
+  focusCbgSlice: (cbgSlice: CbgSlice, position: CbgPositionData<CbgSliceYPositions>, keys: RangeSegmentSlice[]) => void
+  focusedCbgSlice: FocusedCbgSlice | undefined
+  showCbgDateTraces: boolean
   toggleCbgSegments: (displayFlag: DisplayFlag) => void
   unfocusCbgDateTrace: () => void
+  unfocusCbgSlice: () => void
 }
 
+const SHOW_CBG_DATE_TRACES_TIMEOUT_MS = 250
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏I know this is the previous behaviour, but don't you think that we could reduce this duration? It think that it could be more user friendly‏